charm/tests/unit/test_charm.py

```diff
@@ -0,0 +1,129 @@
+# Copyright (C) 2023 Canonical
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+Unit tests for Testflinger Juju charm
+"""
+
+from pathlib import Path
+import pytest
+import yaml
+
+import ops
+import ops.testing
+from charm import TestflingerCharm
+
+CHARMDIR = Path(__file__).parent.parent.parent.resolve()
+METADATA_FILE = CHARMDIR / "metadata.yaml"
+# METADATA = str(yaml.safe_load(Path("metadata.yaml").read_text()))
```

Comment:
should we remove this comment bc it's needed anymore?

Reply:
done, thanks!

Reply:
done, thanks!